micro/goal_test.go

```diff
@@ -57,15 +58,18 @@ func TestNeverO(t *testing.T) {
 }
 
 func TestDisjointO1(t *testing.T) {
+	x := ast.NewVariable("x")
 	d := DisjointO(
 		EqualO(
 			ast.NewSymbol("olive"),
-			ast.NewVariable("x"),
+			x,
 		),
 		NeverO,
 	)()(EmptyState())
 	s, sok := d()
 	got := s.String()
+	// reifying y; we assigned it a random uint64 and lost track of it
+	got = strings.Replace(got, fmt.Sprintf("v%d", x.Atom.Var.Index), "x", -1)
```

Comment:
See the pattern is here again `x.Atom.Var.Index`
It might just be a nice method for an SExpr, that can be used everywhere, instead of having a helper function per test file, what do you think?

Reply:
Ah I missed one. It's only useable on Vars and only being used in tests, I think a one-off helper function is still better

Reply:
Okay, 1 use in the code (reify.go), where I like how explicit it is. Can I keep it, pretty please? :P

Reply:
It just seems as though the tests look a lot like how a user would use it, that is where I am coming from.
But we can also follow up on this at a later time.